_lifeofinternatpuppet/post_7_and_8

```diff
@@ -0,0 +1,186 @@
+
+I was invited to a zoom call by Ciaran and David Schmitt, where they were trying to debug issues on Crontab and its configuration set up.
+
+The error/issue with a solution in Ciaran's own words :-
+
+"Error/Issue: cron is not executing the configured scripts as expected. Permission errors on crontab file being reported to syslog. Additional issue of “can’t fork” errors manifesting this morning
+solution suggested was wrap the command in bash -c to ensure it’s correctly invoked, but the problem still persisted"
+
+During that demo session with Ciaran and David Schmitt, I also asked Ciaran if he had a sync up with Paula or Daiana from yesterday's 1:1 to see if he knows what I did with Paula and Diana/what are the plans for the upcoming week. Ciaran said 'due to other commitments he hasn't had a chance I think, don't remember exactly what he said during the session.
+
+Thats all for now for the week 7 update, happy reading :)
+
+
+Welcome to Week 8...
+
+Time is flying at placement when busy, isn't?
+
+This week a lot is going on within the team, so lets get started...
+
+On Monday 19th October 2020, before having my litmus demo with Norman, I had a quick sync up with Ciaran before the demo at 10:30am.
+During the session with Ciaran he asked me to check if the provisioning of a machine as working fine, because on Saturday when I was doing my practice for the litmus demo, I noticed that the provisioning was failing once again, so that's why the sync up was important with Ciaran prior to the demo.
+We run the process again and it worked fine.
+
+The 1:1 with Norman started off with the demo and session over-ran because there was lots to talk about in relation to tickets being assigned and other important questions...
+
+So I will say firstly that the demo was amazing, I really enjoyed the experience and got more knowledge from someone like Norman who had 20 years experience in mentoring... Sorry Norman, I had to reveal this fact as I think anyone reading this post should be aware of this fact :)
+
+He also said that next time when I am using Litmus and running tests on a provisioned machine, I should select a difficult module/repository to clone down from GitHub, just to give muself a challenge and see if I am do the same process sucessfully!
+I will look into this later in the coming weeks so stay connected to find out more :)
+
+He also suggested that I should read up on the fundamentals of unix and bash commands via [LinkedIn Learning](https://www.linkedin.com/learning/) as it will really help me in my day-to-day work.
+
+He also gave me a breakdown on how I should start my first ticket/task on changing the author name of the supported modules to 'puppetlabs' He gave me useful advice was given on how I should approach the ticket, so thank you so much Norman for all your help to get me started off on my first ticket!ß
```

Comment:
```suggestion
He also gave me a breakdown on how I should start my first ticket/task on changing the author name of the supported modules to 'puppetlabs' He gave me useful advice was given on how I should approach the ticket, so thank you so much Norman for all your help to get me started off on my first ticket!
```

Reply:
So should I make the change and then just push the changes?

Reply:
You can just use the 'Commit suggestion' functionality here

Reply:
There is no & symbol after the word ticket!

Reply:
This suggestion above is to remove the `ß` symbol